zoo_frontend/src/api/Animals.js

```diff
@@ -1,6 +1,10 @@
 import axios from 'axios';
 
-const API_BASE_URL = process.env.BACKEND_URL;
+let API_BASE_URL = process.env.BACKEND_URL;
+
+if (typeof window === 'undefined') {
+  API_BASE_URL = `http://localhost:${process.env.PORT}`;
+}
```

Comment:
would it be better to stick this into a /util file and import this variable?

Reply:
probably